selfservice/hook/web_hook_test.go

```diff
@@ -91,47 +91,78 @@ func TestApiKeyInCookieStrategy(t *testing.T) {
 	assert.Equal(t, "my-api-key-value", cookies[0].Value)
 }
 
+//go:embed stub/test_body.jsonnet
+var testBodyJSONNet []byte
+
 func TestJsonNetSupport(t *testing.T) {
-	f := login.Flow{ID: x.NewUUID()}
+	f := &login.Flow{ID: x.NewUUID()}
 	i := identity.NewIdentity("")
 
-	headers := http.Header{}
-	headers.Add("Some-Header", "Some-Value")
-	headers.Add("Cookie", "c1=v1")
-	headers.Add("Cookie", "c2=v2")
-	data := &templateContext{
-		Flow:           &f,
-		RequestHeaders: headers,
-		RequestMethod:  "POST",
-		RequestUrl:     "https://test.kratos.ory.sh/some-test-path",
-		Identity:       i,
-	}
-
-	b, err := createBody("./stub/test_body.jsonnet", data)
-	assert.NoError(t, err)
-
-	buf := new(strings.Builder)
-	io.Copy(buf, b)
-
-	expected := fmt.Sprintf(`
+	for _, tc := range []struct {
+		desc, template string
+		data           *templateContext
+	}{
+		{
+			desc:     "simple file URI",
+			template: "file://./stub/test_body.jsonnet",
+			data: &templateContext{
+				Flow: f,
+				RequestHeaders: http.Header{
+					"Cookie":      []string{"c1=v1", "c2=v2"},
+					"Some-Header": []string{"Some-Value"},
+				},
+				RequestMethod: "POST",
+				RequestUrl:    "https://test.kratos.ory.sh/some-test-path",
+				Identity:      i,
+			},
+		},
+		{
+			desc:     "filepath without scheme",
+			template: "./stub/test_body.jsonnet",
```

Comment:
Would it make sense to emit a warning if this is used, and at some point deprecate it (and create an issue tracking the deprecation)?

Reply:
I mean we allow it for all other URIs like identity schema as well in the same way...

Reply:
I am actually not sure if we support filepaths in the other things (e.g. jsonnet loader). If we don't I think we should deprecate, if we do, we can keep it

Reply:
Ah right, missed that we have a different loader there. Will mark it as deprecated then.